cmd/api/routes.go

```diff
@@ -41,6 +41,10 @@ func loadPlatforms(root gin.IRouter) {
 		customAPI.RegisterRoutes(router)
 	}
 
+	makeCoinRoute(root)
+	makeCoinRoute(getRouter(v1, ""))
```

Comment:
I think we only need to register one route? v1 is enough, if we need to change something then we add v2

Reply:
root is enough I think